src/ontology/pato-edit.obo

```diff
@@ -21989,6 +21989,78 @@ is_a: PATO:0001655 ! osmolarity
 created_by: http://orcid.org/0000-0002-3410-4655
 creation_date: 2020-10-30T08:08:48Z
 
+[Term]
+id: PATO:0095001
+name: area to mass ratio
```

Comment:
the name is deceptively more general than the definition

e.g. a leaf can have an area to mass ratio